### PR TITLE
[autonity-network] v0.3.6

### PR DESCRIPTION
**Pull request by eastata**
### Changelog:
- Add AWS of GCP persistant storage

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](CONTRIBUTING.md)?
* [x] Is the `branch` name the same as Chart name that you will release?
* [x] Is your changes touch only one chart?
* [x] Is the version of chart incremented in `Chart.yaml`?

Issue #64

### CODE REVIEW

**Comment by Klazomenai** [comment]
Looking good. Can you tag which issues this resolves please.

File: stable/autonity-network/templates/storage_class.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+metadata:
+  name: {{ $releasename }}-validator-{{ $i }}-blockchain
+  namespace: {{ $namespace }}
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: blockchain-{{ $releasename }}
+  resources:
+    requests:
+      storage: 20G
```
Comment (scarby) [inline stable/autonity-network/templates/storage_class.yaml]: should we make this a variable?

File: stable/autonity-network/templates/storage_class.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+metadata:
+  name: {{ $releasename }}-observer-{{ $i }}-blockchain
+  namespace: {{ $namespace }}
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: blockchain-{{ $releasename }}
+  resources:
+    requests:
+      storage: 20G
```
Comment (scarby) [inline stable/autonity-network/templates/storage_class.yaml]: same comment? - Trivial - will approve

File: stable/autonity-network/templates/storage_class.yaml (hunk 1 above)
Comment (eastata) [reply stable/autonity-network/templates/storage_class.yaml]: done

File: stable/autonity-network/templates/storage_class.yaml (hunk 2 above)
Comment (eastata) [reply stable/autonity-network/templates/storage_class.yaml]: done

**Pull request merged**